srcpkgs/cargo/template

```diff
@@ -11,8 +11,8 @@ short_desc="Rust package manager"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="MIT, Apache-2.0"
 homepage="https://crates.io/"
-distfiles="https://github.com/rust-lang/cargo/archive/refs/tags/${_cargo_revision}.tar.gz"
-checksum=c0c15a35852c114859cfe11c5c76ee12105d66d7c90e650af83ac7b8a6a856fc
+distfiles="https://github.com/rust-lang/cargo/archive/${_cargo_commit}.tar.gz"
```

Comment:
https://github.com/rust-lang/cargo/issues/12460 your tag is there now :)

Reply:
Thanks!